lambdas/src/apis/collections/controllers/collections.ts

```diff
@@ -81,7 +78,10 @@ export async function getStandardErc721(client: SmartContentClient, req: Request
     }
     res.send(standardErc721)
   } catch (e) {
-    res.status(e.statusCode ?? 500).send(e.message)
+    if (e instanceof ServiceError) {
+      res.status(e.statusCode).send(e.message)
+    }
+    res.status(500)
```

Comment:
Prevent double .status call
```suggestion
    else {
      res.status(500)
    }
```